events/code_review_events/workflow.py

```diff
@@ -441,7 +403,34 @@ def run(self):
                 # Publish results on Phabricator
                 self.bus.run(self.workflow.publish_results, QUEUE_PHABRICATOR_RESULTS),
                 # Parse and redirect pulse messages
-                self.bus.run(self.workflow.parse_pulse, QUEUE_PULSE),
+                self.workflow.parse_pulse(),
+                self.bus.run(
+                    self.workflow.dispatch_mercurial_applied, QUEUE_MERCURIAL_APPLIED
+                ),
+            ]
+
+            # Publish results on Phabricator
+            if self.workflow.publish:
+                consumers.append(
+                    self.bus.run(
+                        self.workflow.publish_results, QUEUE_PHABRICATOR_RESULTS
+                    )
+                )
```

Comment:
This can now be removed, since we have the publish check in QUEUE_PHABRICATOR_RESULTS.

Reply:
Done in 859d9a1.